tests/pipeline/test_swab_deltas.py

```diff
@@ -0,0 +1,33 @@
+import pandas as pd
+import pytest
+from mimesis.schema import Field
+from mimesis.schema import Schema
+
+from cishouseholds.pipeline.swab_delta_ETL import transform_swabs_delta
+
+_ = Field("en-gb", seed=42)
+
+
+@pytest.fixture
+def input_df(spark_session):
```

Comment:
As fixtures can be called by other test functions, could we name this more informatively so that it's apparent what data the test function is using?

Also, should we store this and other test data in a separate file (e.g. input_dummy_data.py). This could the be imported here to keep the test files clean?

Do you think we're likely to reuse these data in multiple test functions? If not, there's no benefit to using a fixture over a df stored as a variable.

Genuine questions, as I'm not sure which approaches are best 😊

Reply:
Happy to keep data close to test, unless reused elsewhere